frontend/src/scenes/App.tsx

```diff
@@ -53,13 +54,12 @@ function App() {
     }, [scene, user])
 
     if (!user) {
-        return (
-            unauthenticatedRoutes.includes(scene) && (
```

Comment:
We're using this pattern in a lot of places - is this only inappropriate at the top-level due to the return type?

Reply:
yep, `false` is a fine element for React, but this is exported and put into redux's `<Provider>` tag that requires a real element.

The only danger with the `&&` pattern in JSX is that it returns the first element (`false` in this case), but if the value is "falsy" e.g. `0`, then you'll get a random "0" in the DOM. Thus I normally prefer the `? : null` pattern.

Here it was just for the type.

Reply:
Yeah I'm also for the ternary operator. On the website codebase we've been setting classes with `conditional &&` and get `false` and `undefined` as class names rather often.